src/app/charts/line-graph.component.ts

```diff
@@ -115,7 +115,7 @@ export class LineGraphComponent implements OnChanges, AfterContentInit {
 
     /* Will setup the chart basics */
     private setup(): void {
-        this.margin = { top: 10, right: 40, bottom: 30, left: 50 };
+        this.margin = { top: 10, right: 50, bottom: 30, left: 60 };
```

Comment:
Y axis labels can be a lot longer on HDD & CDD than other indicators and got cut off